GitCommands/CommitData.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace GitCommands
 {
-    public class CommitData
+    public sealed class CommitData
```

Comment:
Curious as to why you chose to seal DTO/POCO?
I typically seal behavioural types since I don't like inheritance and can't guarantee that subclasses would not break the expected behaviours. However I leave POCOs open for extension.

Reply:
I tend to seal everything that doesn't have subclasses, even in library code. Once someone uses your API, you can unseal without breaking code, but not the other way around.

I rarely use inheritance when extending capabilities. Composition tends to be more flexible and powerful. When I do use inheritance the base class would have something virtual/abstract most of the time.

While there's no agreed upon definition of POCO, I generally think of a type that doesn't have any real behaviour, nor participate in any inheritance.

Reply:
Thanks.
My experience is different. When you have a single version of software - it may be acceptable to rework the API. However it is a completely different story when you have multiple versions of API consumed by numerous projects...